src/compute/src/sink/persist_sink.rs

```diff
@@ -58,7 +62,13 @@ where
     where
         G: Scope<Timestamp = Timestamp>,
     {
-        let desired_collection = sinked_collection.map(Ok).concat(&err_collection.map(Err));
+        let mut desired_collection = sinked_collection.map(Ok).concat(&err_collection.map(Err));
+
+        // If any non-trivial `REFRESH` options were specified, round up timestamps.
```

Comment:
We should separate the SQL-level concept (`REFRESH`) from the lir-level concept and make it such that it's clear on its own.

Reply:
I've changed the comment.